apps/shinyproxy/templates/configmap.yaml

```diff
@@ -71,4 +71,16 @@ data:
             - mountPath: /etc/shiny-server/shiny-server.conf
               subPath: shiny-server.conf
               name: {{ .Release.Name }}-shiny-configmap
+          {{ if .Values.apps.volumeK8s }}
```

Comment:
This is on the right track but won't this just always mount a volume called data-volume? I think we need to get the volume from the PVC. Something like [this](https://github.com/ScilifelabDataCentre/serve-charts/blob/56a372961bad599cf83579f04fb375155dde1043/apps/custom-app/templates/deployment.yaml#L96). I think the shiny proxy docs show something similar in [this](https://www.shinyproxy.io/documentation/configuration/#kubernetes-1) section.